ros/lib/config.py

```diff
@@ -0,0 +1,40 @@
+import os
+import logging
+
+LOG = logging.getLogger(__name__)
+
+CLOWDER_ENABLED = True if os.getenv("CLOWDER_ENABLED", default="False").lower() in ["true", "t", "yes", "y"] else False
+
+
+def kafka_auth_config(connection_object):
+    if KAFKA_BROKER:
+        if KAFKA_BROKER.cacert:
+            connection_object["ssl.ca.location"] = "/tmp/cacert"
```

Comment:
Regrading module name convention "-" is not allowed and use of "_" (underscores) is discouraged [1]. Will you be OK if I name module rosocp? 
Regarding filename of kafka cacert I don't think it will be required anywhere apart from creating kafka connection object and I think we can make it as constant in future if any such need arise. Let me know your thoughts. 

[1] https://peps.python.org/pep-0008/#package-and-module-names

Reply:
Thanks for the reference on naming convention, I always forget to check that :) . If I understand correctly by below para then underscores are only discouraged in package names? so in that case I would choose ros_ocp,
~~~
Modules should have short, all-lowercase names. Underscores can be used in the module name if it improves readability.
Python packages should also have short, all-lowercase names, although the use of underscores is discouraged.
~~~

For the ca cert file, I can see we referred that twice and so assumed we can have constant for that, sometimes my mind tries to avoid as much errors as we can using the DRY principles :)

Reply:
Done, I have changed the package name and declared kafka ca cert path as constant.